Doc 4078 (#653)
corda/corda-docs-portal: content/en/platform/corda/4.9/enterprise/node/operating/maintenance-mode.md

The following maintenance tasks are currently supported:

- Perform RPC Audit table clean-up
- Run message ID clean-up

## Configuration of Node Maintenance Mode

> Node Maintenance Mode is configured by an optional configuration sub-section named `maintenanceMode` within the `enterpriseConfiguration` top-level [configuration section](../../../../../../../en/platform/corda/4.9/enterprise/node/setup/corda-configuration-fields.html#enterpriseconfiguration).

Reviewer: ```suggestion
You can configure Node Maintenance Mode in an optional configuration sub-section named `maintenanceMode` within the `enterpriseConfiguration` top-level [configuration section](../../../../../../../en/platform/corda/4.9/enterprise/node/setup/corda-configuration-fields.html#enterpriseconfiguration).
```

> By default, no maintenance activities will be performed if the `maintenanceMode` section is not provided. Without this section, Corda behaves as if maintenance mode is not available.

Reviewer: ```suggestion
By default, no maintenance activities are performed if the `maintenanceMode` section is not provided. Without this section, Corda behaves as if maintenance mode is not available.
```
